src/crimson/os/seastore/ordering_handle.h

```diff
@@ -82,6 +82,9 @@ inline OrderingHandle get_dummy_ordering_handle() {
 }
 
 struct WritePipeline {
+  OrderedExclusivePhase throttle{
+    "WritePipeline::throttle_phase"
```

Comment:
I think we do ool writes before the prepare phase, so this may impose additional serialization, am I right?

Reply:
Yep, you are correct.  Will remove that patch from this PR and submit another one for it.